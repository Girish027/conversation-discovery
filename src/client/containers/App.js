import React from 'react';
import { Switch, Route } from 'react-router-dom';
import PageLayout from '../layouts/PageLayout';
import { basePath } from '../swagger.json';
import ErrorBoundary from '../components/ErrorBoundary/ErrorBoundary';


const App = () => (
  <ErrorBoundary>
    <Switch>
      <Route path={basePath} component={PageLayout} />
    </Switch>
  </ErrorBoundary>
);

export default App;
