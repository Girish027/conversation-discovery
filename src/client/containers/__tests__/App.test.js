import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import App from 'containers/App';

describe('<App />', function () {
  let wrapper;

  describe('Snapshots', () => {
    test('renders correctly', () => {
      wrapper = shallow(<App />);
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });
});
