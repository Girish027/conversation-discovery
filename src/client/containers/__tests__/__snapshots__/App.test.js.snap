// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshots renders correctly 1`] = `
<ErrorBoundary>
  <Switch>
    <Route
      component={[Function]}
      path="/conversationdiscovery"
    />
  </Switch>
</ErrorBoundary>
`;
